.github/actions/vtk-install-dep/action.yml

```diff
@@ -93,6 +93,7 @@ runs:
         -DVTK_MODULE_ENABLE_VTK_IOHDF=YES
         -DVTK_MODULE_ENABLE_VTK_IOImage=YES
         -DVTK_MODULE_ENABLE_VTK_IOImport=YES
+        -DVTK_MODULE_ENABLE_VTK_IONetCDF=YES
```

Comment:
line 32 in this file you need to increase the cache value to trigger a cache rebuild with IONetCDF module:

```
key: vtk-${{ inputs.vtk_version }}${{ inputs.openvdb_version != '' && format('-openvdb-{0}', inputs.openvdb_version) || '' }}-${{runner.os}}-${{inputs.raytracing_label}}-${{inputs.cpu}}-1
```